addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -120,4 +123,20 @@
         </t>
     </template>
 
+    <template id="event_booth_progress_bar">
+        <ul class="o_wevent_booth_progress_bar list-unstyled d-none d-lg-block mb-0">
+            <li t-attf-class="position-relative float-start m-0 text-center #{'o_current_stage' if step=='STEP_BOOTH_SELECTION' else ''}">
+                <a class="text-decoration-none text-reset" t-attf-href="/event/#{slug(event)}/booth?#{keep_query('booth_category_id', 'booth_ids')}">
+                    <span class="d-block">Booth Selection</span><span class="chevron position-absolute d-block"/>
+                </a>
+            </li>
+            <li t-attf-class="position-relative float-start m-0 text-center #{'o_current_stage' if step=='STEP_DETAILS_FORM' else 'text-muted'}">
+                Contact Details<span class="chevron position-absolute d-block"/>
+            </li>
+            <li class="position-relative float-start m-0 text-center text-muted">
```

Comment:
Not in this template. There is a qweb template called from the JS for that case (free + confirmed). (event_booth_registration_complete, only for the confirmation stage, where o_current_stage is indeed used). If not free, we go to the sale process instead of first stage. The one here is used for the other cases.